.gitignore

```diff
@@ -13,6 +13,7 @@ parts
 docs/_build
 htmlcov/
 venv/
+env/
```

Comment:
what is this?

Reply:
It's python virtual environment directory just like `venv`

Reply:
But it is your virtualenv, that is only your configuration. You can just use the pre-defined `venv` instead of bring a new config.

Reply:
you're right. I'll fix it